src/handlers/expenditures/helpers/decodeSetExpenditureState.ts

```diff
@@ -50,7 +46,7 @@ export const decodeUpdatedSlots = (
         recipientAddress,
       });
     } else if (keys[1] === EXPENDITURESLOT_CLAIMDELAY) {
-      const claimDelay = toNumber(value);
+      const claimDelay = toNumber(value).toString();
```

Comment:
I assume we convert these to strings so that we can save them via mutations, right ?